AIO/src/org/aio/script/AIO.java

```diff
@@ -23,7 +23,7 @@
 import java.util.Optional;
 import java.util.Queue;
 
-@ScriptManifest(author = "Explv", name = "Explv's AIO v1.7", info = "AIO", version = 1.7, logo = "http://i.imgur.com/58Zz0fb.png")
+@ScriptManifest(author = "Explv", name = "Explv's AIO v2.0", info = "AIO", version = 2.0, logo = "http://i.imgur.com/58Zz0fb.png")
```

Comment:
Everything looks perfect, version should be 1.8 though (one above the live version)

Reply:
Fixed